ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AVD_LAB.yml

```diff
@@ -32,6 +32,10 @@ name_servers:
   - 192.168.200.5
   - 8.8.8.8
 
+# Management api-http
+management_eapi:
+  default_services: false
```

Comment:
Please also set `enable_https: true` since you are overriding the default value. Otherwise it will produce the changes noted in my other comment.

Reply:
I will update the https knob also.